src/Abc.Zebus.Tests/Core/BusTests.Core.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Abc.Zebus.Core;
+using Abc.Zebus.Directory;
+using Abc.Zebus.Routing;
+using Abc.Zebus.Testing;
+using Abc.Zebus.Testing.Comparison;
+using Abc.Zebus.Testing.Extensions;
+using Abc.Zebus.Testing.UnitTesting;
+using Abc.Zebus.Tests.Messages;
+using Abc.Zebus.Transport;
+using Moq;
+using NUnit.Framework;
+
+namespace Abc.Zebus.Tests.Core
+{
+    public partial class BusTests
```

Comment:
Please commit these kind of refactorings in a separate commit next time to make the relevant changes more easy to find.